crates/scarb-artifacts/src/lib.rs

```diff
@@ -87,7 +87,7 @@ pub fn try_get_starknet_artifacts_path(
 ///
 /// # Arguments
 ///
-/// * path - A path to the Scarb package
+/// * `path`` - A path to the Scarb package
```

Comment:
It is not a path to the package, but to `target_name.starknet_artifacts.json` file, idk why it was like this before 🤔